frontend/pages/policies/ManagePoliciesPage/components/PoliciesTable/PoliciesTableConfig.tsx

```diff
@@ -99,12 +104,29 @@ const generateTableHeaders = (
     canAddOrDeletePolicy?: boolean;
     tableType?: string;
   },
-
+  policiesList: IPolicyStats[] = [],
   isPremiumTier?: boolean,
   isSandboxMode?: boolean
 ): IDataColumn[] => {
   const { selectedTeamId, tableType, canAddOrDeletePolicy } = options;
 
+  // Figure the time since the host counts were updated.
+  // First, find first policy item with host_count_updated_at.
+  const updatedAt =
+    policiesList.find((p) => !!p.host_count_updated_at)
+      ?.host_count_updated_at || "";
+  let timeSinceHostCountUpdate = "";
+  if (updatedAt) {
+    try {
+      timeSinceHostCountUpdate = formatDistanceToNowStrict(
```

Comment:
We'll be able to count on `host_count_updated_at` being a valid date right?

Reply:
Yes, null or a valid date.